transaction/core/src/ring_signature/rct_bulletproofs.rs

```diff
@@ -292,16 +292,24 @@ fn sign_with_balance_check<CSPRNG: RngCore + CryptoRng>(
     };
 
     if check_value_is_preserved {
-        let sum_of_output_commitments: RistrettoPoint = output_values_and_blindings
+        let sum_of_output_values: u64 = output_values_and_blindings
```

Comment:
This could experience overflow

Reply:
yeah -- i think in the tests it does, and in the other branch, i was converting to scalar first. thanks